backend/backend/queries/lost_and_found.py

```diff
@@ -0,0 +1,110 @@
+from pypika import Query, Table, functions as fn, Order
+from typing import Dict, Any
+from enum import Enum
+
+lost_table, lost_images_table = Table('lost'), Table('lost_images')
+found_table, found_images_table = Table('found'), Table('found_images')
+users = Table("users")
+
+class TableType(Enum):
+    LOST = lost_table
+    FOUND = found_table
+class TableImagesType(Enum):
+    LOST = lost_images_table
+    FOUND = found_images_table
+
```

Comment:
It's nice that you used enums. For uniformity, could you move these enums to `models.py` and make appropriate imports?